script/lib/util.py

```diff
@@ -147,6 +161,12 @@ def make_zip(zip_file_path, files, dirs):
         zip_file.close()
 
 
+def make_7z(archive_file_path, files, dirs):
+    safe_unlink(archive_file_path)
+    files += dirs
+    execute([get_lzma_exec(), 'a', '-t7z', archive_file_path] + files)
```

Comment:
Do we want to pass `-mx` for a higher compression level?

Reply:
```
-mx4 time: 0:00:21.630766 (256.664%), size: 649.992MB (98.4123%)
-mx5 time: 0:01:07.924675 (314.019%), size: 574.392MB (88.3691%)
-mx6 time: 0:01:19.725151 (117.373%), size: 570.369MB (99.2996%)
-mx7 time: 0:01:31.076979 (114.239%), size: 551.987MB (96.7772%)
-mx8 time: 0:02:47.909305 (184.36%),  size: 543.338MB (98.433%)
-mx9 time: 0:02:48.007341 (100.241%), size: 543.338MB (100%)
```
Default is `-mx5`, I think we'll be fine with `-mx7`. `-mx8` almost doubles the time without noticeable size change.